system/EntityCast/CastAsTimestamp.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+namespace CodeIgniter\EntityCast;
+
+/**
+ * Class CastAsTimestamp
+ *
+ * @package CodeIgniter\Entity\Cast
+ */
+
+class CastAsTimestamp extends AbstractCast
+{
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get($value, array $params = [])
+	{
+		return strtotime($value);
```

Comment:
Maybe you can additionally check the return value if not `false`. If `false`, then throw an error for invalid timestamp.